src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/types/BeanstreamEndpointResponse.java

```diff
@@ -0,0 +1,69 @@
+package ca.bc.gov.open.pssg.rsbc.dps.paymentservice.types;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class BeanstreamEndpointResponse {
+
+    private String approved;
+    private String declined;
+    private String error;
+    private String respMsg;
+    private int respCode;
+
+    public BeanstreamEndpointResponse( String approved, String declined, String error, String respMsg, int respCode){
+        this.approved = approved;
+        this.declined = declined;
+        this.error =error;
+        this.respMsg = respMsg;
+        this.respCode = respCode;
+    }
+
+    public String getRespMsg() {
+        return respMsg;
+    }
+
+    public void setRespMsg(String respMsg) {
```

Comment:
we should remove setters as we are using the constructor which is the right approach

Reply:
removed the setters